editor/editor_themes.cpp

```diff
@@ -1144,11 +1144,8 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_constant("label_width", "ColorPicker", 10 * EDSCALE);
 	theme->set_icon("screen_picker", "ColorPicker", theme->get_icon("ColorPick", "EditorIcons"));
 	theme->set_icon("add_preset", "ColorPicker", theme->get_icon("Add", "EditorIcons"));
-	theme->set_icon("preset_bg", "ColorPicker", theme->get_icon("GuiMiniCheckerboard", "EditorIcons"));
```

Comment:
The icons "ColorPicker":"preset_bg" and "ColorPickerButton":"bg" are just references to the icon "EditorIcons":"GuiMiniCheckerboard". I'm removing the middle-man for now.

Reply:
You can't remove them. It breaks ColorPickers used outside editor, they don't have access to editor icons. And also this makes them impossible to customize by the users.

Reply:
Added them back. Also added the same thing for GradientEdit since previously it was hard coded in :)

Reply:
GradientEdit doesn't need it though, because it's not exposed class.